test/Application/BadControllerTrait.php

```diff
@@ -72,7 +73,7 @@ public function prepareApplication()
                     },
                 ],
                 'invokables' => [
-                    'Request'              => 'Zend\Http\PhpEnvironment\Request',
+                    'Request'              => Request::class,
```

Comment:
The topic of this PR is "PHPUnit 6" and all these changes makes a review more difficult.